### PR TITLE
Reviews ui

### PR DESCRIPTION
**Pull request by az9222**
Server completed
Basic react rendered

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -26,10 +27,22 @@ class App extends React.Component {
     .catch(() => {console.log("Error: Reviews not found.")}
```
Comment (TonyWu5) [inline client/src/components/App.jsx:28]: not clear why you're trying to catch an error here.

File: client/src/components/App.jsx [hunk 2]
```diff
@@ -26,10 +27,22 @@ class App extends React.Component {
...
   )};
 
+  renderReviews() {
+    return (
+      <Reviews sampleReviews={this.state.reviews}/>
```
Comment (TonyWu5) [inline client/src/components/App.jsx]: "sampleReviews" can be the value of your this.state.reviews, but I'd pass it into the components with a different variable name (just "reviews").
this is important when you're app is fully functional, when you're component is no longer using sample data.

File: client/src/components/NoReviews.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const NoReviews = (props) => (
```
Comment (TonyWu5) [inline client/src/components/NoReviews.jsx:3]: maybe props don't need to be passed here?

File: client/src/components/Reviews.jsx [hunk 4]
```diff
@@ -9,14 +9,26 @@ const Reviews = (props) => (
             <span>Dined on </span><span>{sampleReview.review_time}</span>
```
Comment (TonyWu5) [inline client/src/components/Reviews.jsx]: do you need to reference the data as {props.sampleReviews.review_time}...I could be wrong.

File: server/index.js [hunk 5]
```diff
@@ -57,13 +57,14 @@ app.post('/reviews', (req, res) => {
       overall_rating: Joi.number().min(0).max(5).required(), 
       food_rating: Joi.number().min(0).max(5).required(), 
       service_rating: Joi.number().min(0).max(5).required(),
```
Comment (TonyWu5) [inline server/index.js:59]: I agree with Alex, this should probably reside on the client side.

File: client/src/components/App.jsx (hunk 1 above)
Comment (az9222) [reply client/src/components/App.jsx:28]: In case the promise doesn't fetch anything. Do you suggest something else?

**Pull request merged**